### DISCUSSION TITLE
NuGet Package unlisted

### DISCUSSION THREAD

**Discussion by RouvenRo** [UiPath/corewf#312] [Q&A]
We've encountered a discrepancy regarding the CoreWF NuGet package and would appreciate any insights or information.

On the NuGet [website](https://www.nuget.org/packages/UiPath.Workflow), the CoreWF package is unlisted: "The owner has unlisted this package. This could mean that the package is deprecated, has security vulnerabilities, or shouldn't be used anymore." However, we've observed ongoing development activity in the repository here, suggesting that development may be continuing.

We are keen to understand the current status of the CoreWF package. Can anyone shed light on reasons for its unlisted status?

**Comment by ivan-meling-delinea** [comment]
We're trying to migrate some old .NET Framework projects to .NET 10, this library would be necessary. Can someone from the development team answer this question?
